src/apps/frontend/components/flex/flex-item.component.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from "react";
+
+interface FlexItemProps {
+  children: ReactNode;
+  className?: string;
```

Comment:
This is not a good design pattern, this means the consumer can change the layout, style of this anyway they choose. Breaks abstraction pretty quickly

Reply:
This is fixed.
Added all the properties that we can give to a flexItem